api/src/identity-access-management/domain/usecases/authenticate-oidc-user.usecase.js

```diff
@@ -87,18 +87,40 @@ async function authenticateOidcUser({
     });
   }
 
+  await _updateUserLastConnection({
+    user,
+    oidcAuthenticationService,
+    authenticationMethodRepository,
+    lastUserApplicationConnectionsRepository,
+    requestedApplication,
+    userLoginRepository,
+  });
+
+  return { pixAccessToken, logoutUrlUUID, isAuthenticationComplete: true };
+}
+
+export { authenticateOidcUser };
+
+async function _updateUserLastConnection({
+  user,
+  oidcAuthenticationService,
+  authenticationMethodRepository,
+  lastUserApplicationConnectionsRepository,
+  requestedApplication,
+  userLoginRepository,
+}) {
```

Comment:
Il me semble qu'il est plus clair de mettre toutes les dependencies à la fin, même si évidemment l'ordre de déclaration dans la object literal syntax n'a pas d'importance.

```suggestion
async function _updateUserLastConnection({
  user,
  requestedApplication,
  oidcAuthenticationService,
  authenticationMethodRepository,
  lastUserApplicationConnectionsRepository,
  userLoginRepository,
}) {
```

Je ne mets pas ce commentaire dans tous les usecases mais l'idée est de faire la modif dans tous les usecases.